docs/recipes/bitbucket-pipelines.md

```diff
@@ -4,48 +4,101 @@
 
 The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured in [Pipelines Repository Settings](https://confluence.atlassian.com/bitbucket/variables-in-pipelines-794502608.html)
 
+## Node project configuration
 
-### Node.js projects configuration
-install all the required plugins
-```
-npm i -D @semantic-release/changelog@next @semantic-release/commit-analyzer@next @semantic-release/git@next @semantic-release/npm@next @semantic-release/release-notes-generator@next semantic-release@next
+[Bitbucket Pipelines](https://confluence.atlassian.com/bitbucket/use-docker-images-as-build-environments-in-bitbucket-pipelines-792298897.html) can be configured to run any container from Dockerhub or private registries
+
+**Note**: The publish pipeline must run on [Node version >= 10.18](../support/FAQ.md#why-does-semantic-release-require-node-version--1018).
+
+Install all the required plugins (including the generic `git` plugin)
+
+```bash
+npm i -D @semantic-release/changelog @semantic-release/commit-analyzer @semantic-release/git @semantic-release/npm @semantic-release/release-notes-generator semantic-release
 ```
 
-### Package.json configuration
+### 'semantic-release' configuration
+
 Make sure you configure your package.json file with the required plugins.
 A sample configuration may look like this.
 
-```
+```json
 "devDependencies": {
-    ...
-  },
+  ...
+},
 "release": {
   "plugins": [
-    "@semantic-release/commit-analyzer",
-    "@semantic-release/release-notes-generator",
-    "@semantic-release/npm",
-    "@semantic-release/changelog",
-    "@semantic-release/git"
+  "@semantic-release/commit-analyzer",
+  "@semantic-release/release-notes-generator",
+  "@semantic-release/npm",
+  "@semantic-release/changelog",
+  "@semantic-release/git"
   ]
-},
-  ```
+}
+```
+
+### `bitbucket-pipelines.yml` configuration
+
+You need to allow 2 things for this workflow to work with Bitbucket:
+- authorize the pipeline to [push back to the repository](https://support.atlassian.com/bitbucket-cloud/docs/push-back-to-your-repository/): it should work seamlessly by default if your repository uses the HTTPS origin. If you use the SSH origin for any reason, allow access with your pipeline SSH keys
+- authorize the pipeline to publish to npm (you will need a read+write key from npmjs.com)
 
-### `bitbucket-pipelines.yml` Configuration
 1. Make sure you have the required environment variables like NPM_TOKEN and/or NPM_REGISTRY_URL set up correctly in the repository's pipelines' settings from bitbucket's UI.
 2. create a `bitbucket-pipelines.yml` file in the root directory of the repository with the following.
 
-```
-image: node:latest
+```yaml
+# bitbucket-pipelines.yml
+# You can use any image & tag, but node >= 10.18 is required
+image: node:erbium
```

Comment:
is erbium required in specific? maybe we can set it to `lts` or `lts-alpine` to make sure its up to date with the latest LTS version when they switch to node v14+

Reply:
No, the only requirement for semantic-release is `node >= 10.18` at the moment because this is their minimal supported version :)

I was not even aware of the `lts` tag, this is probably better to use in the doc then, I will update it :)

Regarding alpine, the only "issue" is that `npm install` (or `npm ci`) could fail on alpine if the image does not provide some dependencies for some packages such as `imagemin` for instance.
If would rather mention it but maybe not propose it as default?
(Anyone is free to set their preferred image anyway)

Reply:
I have just changed it to `node:lts` and added mentions of alpine

Reply:
looks good, thanks